lib/utils/circuit-json/inflate-circuit-json.ts

```diff
@@ -81,7 +81,50 @@ export const inflateCircuitJson = (
   }
 
   const sourceTraces = injectionDb.source_trace.list()
+  const pcbTraces = injectionDb.pcb_trace.list()
+
+  const pcbTracesBySourceTraceId = new Map<string, typeof pcbTraces>()
+  for (const pcbTrace of pcbTraces) {
+    const sourceTraceId = pcbTrace.source_trace_id as string | undefined
+    if (!sourceTraceId) continue
+    const list = pcbTracesBySourceTraceId.get(sourceTraceId) ?? []
+    list.push(pcbTrace)
+    pcbTracesBySourceTraceId.set(sourceTraceId, list)
+  }
+
```

Comment:
Naming unclear. There are two types of ids, some from the inflation db and others from the circuit, so you need to prefix. Also this datastructure is unnecessary, just use the inflation db

Reply:
Dont create id maps because more variables adds complexity for little gain